migrations/20220502034714_campaign_file_column_update/migration.sql

```diff
@@ -0,0 +1,16 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `uploaded_by_id` on the `campaign_files` table. All the data in the column will be lost.
+  - Added the required column `person_id` to the `campaign_files` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- DropForeignKey
+ALTER TABLE "campaign_files" DROP CONSTRAINT "campaign_files_uploaded_by_id_fkey";
+
+-- AlterTable
+ALTER TABLE "campaign_files" DROP COLUMN "uploaded_by_id",
+ADD COLUMN     "person_id" UUID NOT NULL;
```

Comment:
Let's change this migration manually to a SQL `RENAME` so we don't loose the data in `uploaded_by_id` column or copy the values manually (Add new column, copy values, delete old column)